site/guide/2-getting-started.md

```diff
@@ -81,13 +81,13 @@ run`. You should see the following:
     => address: localhost
     => port: 8000
     => log: normal
-    => workers: [logical cores * 2]
+    => workers: 16
```

Comment:
This (and the other one) should be `[logical cores * 2]`, to keep it generic.

Reply:
Suggestion was accepted.

Reply:
Changes have been accepted. Thanks for the info!